recipe/meta.yaml

```diff
@@ -27,7 +27,8 @@ requirements:
   run:
     - cvxopt >=1.2.0
     - h5py >=2.10.0
-    - matplotlib-base >=3.1.1
+    - napari >=0.4.16
+    - napari-plugin-engine
```

Comment:
Scipy does not seem to be used upstream, but it still listed in the requirements @jaimergp.